frontend/components/site/SiteSettings/CopyRepoForm.jsx

```diff
@@ -23,18 +23,19 @@ class CopyRepoForm extends React.Component {
   renderDisclaimer() {
     return (
       <p>
-        This feature will allow you to copy this site to a new GitHub repository.
-        You must be an admin user of this site.
+        This feature will allow you to copy this site's code to a new GitHub repository.
```

Comment:
`'` in `site's` should be `&apos;`: `site&apos;s`

Reply:
ohhh that makes more sense